packages/web/e2e/uploadTrack.test.ts

```diff
@@ -0,0 +1,224 @@
+import { expect } from '@playwright/test'
+import {
+  EditTrackPage,
+  FinishPage,
+  SelectPage
+} from './page-object-models/upload'
+import {
+  RemixSettingsModal,
+  AccessAndSaleModal,
+  AttributionModal,
+  StemsAndDownloadsModal
+} from './page-object-models/modals'
+import { test } from './test'
+
+test('should upload a remix, hidden, AI-attributed track', async ({ page }) => {
+  const trackTitle = `Test track ${Date.now()}`
+  const trackDescription = 'Test description'
+  const genre = 'Alternative'
+  const mood = 'Easygoing'
+  const tags = ['TAG1', 'TAG2']
+  const remixUrl = 'staging.audius.co/sebastian12/probers_track_do_not_delete'
+  const remixName = 'probers_track_do_not_delete'
+  const aiAttributionName = 'probers ai DO NOT DELETE'
+  const isrc = 'US-123-45-67890'
+  const iswc = 'T-123456789-0'
+
+  await page.goto('upload')
+
+  const selectPage = new SelectPage(page)
+  await selectPage.setTracks('track.mp3')
+  await selectPage.continue()
+
+  await expect(
+    page.getByRole('heading', { name: /complete your track/i, level: 1 })
+  ).toBeVisible()
+
+  const editPage = new EditTrackPage(page)
+  await editPage.setArtwork('track-artwork.jpeg')
+  await editPage.setTitle(trackTitle)
+  await editPage.setDescription(trackDescription)
+  await editPage.setGenre(genre)
+  await editPage.setMood(mood)
+  await editPage.setTags(tags)
+
+  await editPage.openRemixSettings()
+  const remixSettingsModal = new RemixSettingsModal(page)
+  await remixSettingsModal.setAsRemixOf(remixUrl, remixName)
+  await remixSettingsModal.save()
```

Comment:
idea: what if `.openRemixSettings()` could return the modal object? Since its an "output" of the `openRemixSettings` action anyways
```
const remixSettingsModal = await editPage.openRemixSettings()
```

Reply:
I thought about this and initially wrote it this way, but I think that's coupling a bit too much and I felt like keeping the POMs fairly "dumb" was a better play